index.html

```diff
@@ -8788,7 +8760,11 @@ <h3>Graph Containers</h3>
       <a data-cite="HTML52/semantics-scripting.html#the-script-element">script element</a>
       within an HTML document may be located using
       a fragment identifier matching the <a data-cite="DOM#concept-id">unique identifier</a>
-      of the script element within the HTML document located by a URL (see [[!DOM]]).</p>
+      of the script element within the HTML document located by a URL (see [[!DOM]]).
+      A <a>JSON-LD processor</a> MUST extract only the specified data block's contents
+      parsing it as a standalone <a>JSON-LD document</a>
+      and MUST NOT create a <a>dataset</a>.</p>
```

Comment:
Why must it not create a dataset; what does it create if not a dataset?

Reply:
I would want a targeted extraction to extract exactly what's expressed in the data block, and not alter it by putting it within a dataset.

If I'm extracting all data blocks into a single "thing", then I'd expect that thing to be a dataset.

Reply:
Okay, wording needs to be changed, as the embedded JSON-LD may, itself, create a dataset, which should not be prohibited. I think you mean that an quads extracted from a targeted script should _not_ be combined into a graph/dataset comprised of other markup in the document, including other JSON-LD script elements, embedded Mircodata, RDFa, or whatever.

Reply:
Commit 63017bd8ced68fd27e949281c1821e2ce4482bcd in this branch should resolve this, and I think we can merge.